collectd master fails to build on OSX

```
  CC       collectd_tg-collectd-tg.o
../../collectd/src/collectd-tg.c:107:7: error: implicit declaration of function 'clock_gettime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  if (clock_gettime (CLOCK_MONOTONIC, &ts) != 0)
      ^
../../collectd/src/collectd-tg.c:107:22: error: use of undeclared identifier 'CLOCK_MONOTONIC'
  if (clock_gettime (CLOCK_MONOTONIC, &ts) != 0)
                     ^
2 errors generated.
make[3]: *** [collectd_tg-collectd-tg.o] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all] Error 2
make: *** [all-recursive] Error 1
```

@octo Thanks for the fix, but you missed the declaration of gettimeofday:

```
collectd-tg.c:119:7: error: implicit declaration of function 'gettimeofday' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  if (gettimeofday (&tv, /* timezone = */ NULL) != 0)
      ^
1 error generated.
make[3]: *** [collectd_tg-collectd-tg.o] Error 1
```

--- a/src/collectd-tg.c
+++ b/src/collectd-tg.c
@@ -40,6 +40,7 @@
 #include <signal.h>
 #include <errno.h>
 #include <math.h>
+#include <sys/time.h>
 
 #include "utils_heap.h"
 

